db/migrate/20220727083528_res05_foodvalley.rb

```diff
@@ -1,6 +1,6 @@
 class Res05Foodvalley < ActiveRecord::Migration[5.0]
```

Comment:
Why did you rename an old migration instead of creating a new one? Was the old data bad? It will still remain in the database commits this way.